refresh.py

```diff
@@ -483,38 +679,44 @@ def write_out_results(item, item_name: str):
         raise Exception(f"Unknown type {type(item)}")
 
 
-def load_results(data_path):
+def load_results(data_path: str) -> list | dict | pd.DataFrame | str | None:
```

Comment:
It might be worth simplifying the type here

Reply:
Agree. At some point it would be nice to go through the results files and only keep what we actually need. It seems like there is some redundancy but I haven't had time to go through carefully.  Then we could load just a dataframe or something.